### PR TITLE
Fix search conversation issue

### PR DESCRIPTION
**Pull request by DigneshGujarathi**

### CODE REVIEW

File: src/containers/Chat/ChatMessages/ChatMessages.tsx [hunk 1]
```diff
@@ -344,6 +359,15 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId }) => {
     updateConversationsCache(allConversationsCopy, client, queryVariables);
   };
 
+  // conversationInfo should not be empty
+  if (!Object.prototype.hasOwnProperty.call(conversationInfo, 'contact')) {
```
Comment (rathorevaibhav) [inline src/containers/Chat/ChatMessages/ChatMessages.tsx:363]: I think you can also do this
```suggestion
  if (!conversationInfo.hasOwnProperty('contact')) {
```

**Review by rathorevaibhav — COMMENTED** [review]
Overall looking good. Just left one minor comment.

File: src/containers/Chat/ChatMessages/ChatMessages.tsx (hunk 1 above)
Comment (DigneshGujarathi) [reply src/containers/Chat/ChatMessages/ChatMessages.tsx:363]: It has been updated by the eslint rules, please review - https://eslint.org/docs/rules/no-prototype-builtins

Comment (rathorevaibhav) [reply src/containers/Chat/ChatMessages/ChatMessages.tsx:363]: Makes sense. Looking good.

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**